pkg/logbridge/logbridge.go

```diff
@@ -100,8 +129,9 @@ func (lb *LogBridge) lossyCopy(r io.Reader, lines chan []byte) {
 		case lines <- line:
 		default:
 			droppedLines++
+			lb.droppedLineCount.Inc(1)
```

Comment:
Discussed IRL, will switch to `ReserveN` in order to track a second gauge related to how long we were waiting